.github/workflows/spec14-qa-test-suite.yml

```
@@ -200,7 +200,7 @@ jobs:
   # Performance and coverage validation
   validate-coverage:
     needs: [test-core]
-    if: always() && needs['pre-flight'].outputs['should-run-full-suite'] == 'true'
+    if: always() && needs['pre-flight'].outputs['should_run_full_suite'] == 'true'
```

actionlint (1.7.7)

property "pre-flight" is not defined in object type {test-core: {outputs: {}; result: string}}

(expression)